compiler-rt/lib/tsan/rtl/tsan_report.cpp

```diff
@@ -103,12 +105,23 @@ void PrintStack(const ReportStack *ent) {
     Printf("    [failed to restore the stack]\n\n");
     return;
   }
+
+  if (common_flags()->enable_symbolizer_markup) {
+    if (const ListOfModules *modules =
```

Comment:
I think you changed the code to never return nullptr here, so this if doesn't do anything?